packages/types/src/oauth.ts

```diff
@@ -106,9 +108,15 @@ export const OAUTH_PROVIDERS: OAuthProviderData[] = [
   {
     provider: 'linkedin',
     strategy: 'oauth_linkedin',
-    name: 'LinkedIn',
+    name: 'LinkedIn (Deprecated)',
```

Comment:
❓ @fragoulis Unless I'm mistaken, this is used by clerk-js as well. Is this visile to the end users as well?

Reply:
Yes, this is what is shown in the Dashboard. See [here](https://github.com/clerkinc/dashboard/pull/1636). Do you think there would be a problem somewhere else?

Reply:
@fragoulis 
Unfortunately, it seems that this string is being used by the UI. Do we need the deprecated comment here? 
![image](https://github.com/clerkinc/javascript/assets/1811063/d2b66d8d-c5f1-4685-8738-b89796ef4fd0)

To share a little bit of context, It looks like this type and the utils that exist here are shared between the dashboard and the Clerk sdks. The `(deprecated)` string needs to be moved in the dashboard. We also should discuss how we want to approach this in the future with the dashboard team, as it seems that the types package includes actual runtime code like `getOAuthProviderData` and `sortedOAuthProviders` for the sole purpose of code reuse between the sdks and the dashboard. I believe, however, that the sdks should not be aware of the dashboard at all.

Happy to talk with you and Mark about this when you have some free time next week :)

Reply:
This feels completely out of my comfort zone.

Reply:
![image](https://github.com/clerkinc/javascript/assets/1748736/916fd2e3-23a1-4a60-a094-430f84193bdc)

![image](https://github.com/clerkinc/javascript/assets/1748736/9a96e94c-d6b6-49e7-a5f3-b3694984b6ee)

These are the other places it is shown.

Reply:
I think @nikosdouvlis is right. We don't want to display the "(Deprecated)" part in our components. We should probably create a new provider with the same name as the old one.

Something like the following
```js
{
  provider: "linkedin",
  strategy: "oauth_linkedin",
  name: "LinkedIn",
},
{  
  provider: "linkedin_oidc",
  strategy: "oauth_linkedin_oidc",
  name: "LinkedIn",
}
```

But we should make sure that only one of the two providers can be enabled.